### PR TITLE
Run helix tests locally

### PR DESCRIPTION
**Pull request by BrennanConroy**
* Making it easier to run local test projects on helix
* Fixing bug with running quarantined tests on Windows

### CODE REVIEW

**Review by HaoK — APPROVED** [review]
Nice!

File: eng/helix/content/runtests.cmd [hunk 1]
```diff
@@ -23,7 +23,7 @@ powershell.exe -NoProfile -ExecutionPolicy unrestricted -Command "[Net.ServicePo
 
 set HELIX=%helixQueue%
 
-if (%targetFrameworkIdentifier%==.NETFramework) (
+if %targetFrameworkIdentifier%==.NETFramework (
```
Comment (BrennanConroy) [inline eng/helix/content/runtests.cmd]: Is this just old code that we should get rid of?

**Comment by HaoK** [comment]
I thought it was but when I deleted it there is one test that still runs on net461 shrug. So I added it back

**Comment by anurse** [comment]
@BrennanConroy can you send an email out pointing out this doc and how to run tests in Helix "locally" once this merges?

**Comment by BrennanConroy** [comment]
Yep, that's what I was planning on doing

**Pull request merged**